src/UserList.js

```diff
@@ -1,20 +1,21 @@
 import React from "react";
 import users from "./getUsers";
 
-const v = { display: "block" };
-const h = { display: "none" };
+const v = {display: "block"};
+const h = {display: "none"};
 
 export default class UserList extends React.Component {
-    state = {
-        clicked: 0
-    };
-    onClickEmitter = e => {
+    // state = {
+    //     clicked: 0
+    // };
+
+    onClickEmitter = (userName) => {
         for (let i = 0; i < users.length; i++) {
-            if (users[i].name !== e.target.getAttribute("data-name")) {
```

Comment:
This is more like an anti-pattern for react to set custom data attributes on elements and read them back. The better solution as I modified here is to send the username from the click handler using an anonymous method (line 33)